### DISCUSSION TITLE
SSLKEYLOGFILE on windows

### DISCUSSION THREAD

**Discussion by glslang** [curl/curl#13665] [Q&A] [answered]
Hi,

Is SSLKEYLOGFILE supported on windows? Using the latest official build I always get an empty file. Is there something else to enable or is it the case the build doesn't fully enable it? And if not is it easy enough to enable it and provide an official build with it?

curl 8.7.1 (x86_64-w64-mingw32) libcurl/8.7.1 LibreSSL/3.9.2 zlib/1.3.1 brotli/1.1.0 zstd/1.5.6 WinIDN libpsl/0.21.5 libssh2/1.11.0 nghttp2/1.62.0 ngtcp2/1.5.0 nghttp3/1.3.0
Release-Date: 2024-03-27
Protocols: dict file ftp ftps gopher gophers http https imap imaps ipfs ipns ldap ldaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp ws wss
Features: alt-svc AsynchDNS brotli HSTS HTTP2 HTTP3 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM PSL SPNEGO SSL SSPI threadsafe UnixSockets zstd

Many thanks

**Comment by bagder** [comment]
I am not aware of any reasons why it should not work on Windows but I have never tried it myself.

**Comment by jay** [comment, marked as answer]
> Is SSLKEYLOGFILE supported on windows? Using the latest official build I always get an empty file.

That's a bug, please take further discussion to #13672
